prdoc/pr_4375.prdoc

```diff
@@ -0,0 +1,45 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Support different assets for delivery fees
+
+doc:
+  - audience: Runtime User
+    description: |
+      We now support different assets for delivery fees.
+      This means reserve asset transfers from one parachain to another going through AssetHub
+      no longer need DOT/KSM to pay for delivery fees on AssetHub.
+      They work as long as there is any asset which has a pool with DOT/KSM on AssetHub.
+      Right now paying delivery fees with a different asset locally is not supported, but it will
+      be in the future.
+      This PR supports different assets for remote delivery fees.
+  - audience: Runtime Dev
+    description: |
+      In order to support different assets for delivery fees, the existing AssetExchanger config item
+      was leveraged.
+      This config item allows exchanging assets and quoting exchange prices.
+      An adapter was added to `xcm-builder` to make it compatible with `pallet-asset-conversion`:
+      `FungiblesPoolAdapter`.
+      The new functionality in the executor pays delivery fees using only one asset, which might be
+      swapped to the correct one.
+      If you have a custom XcmRouter that returns more than one asset needed for delivery fees, then know
+      that only the first one will be used.
```

Comment:
I look at the doc, it seems ambiguous about what does `Assets` means when returned as `SendResult`.

Maybe it should be specified there that `Assets` as returned in `SendResult` means only the first asset will be used as payement.